phpunit.xml.dist

```diff
@@ -24,4 +24,7 @@
             </exclude>
         </whitelist>
     </filter>
+    <php>
+        <env name="SYMFONY_DEPRECATIONS_HELPER" value="weak" />
```

Comment:
it should not always be weak. We precisely want to prevent using deprecated APIs (users will complain if you use such API).
You should fix the usage of deprecated APIs instead